openerp/addons/base/ir/ir_qweb/ir_qweb.py

```diff
@@ -148,7 +145,7 @@ def _compile_directive_call_assets(self, el, options):
 
     @tools.conditional(
```

Comment:
add a one-line comment to explain that in non-xml-debug mode we want assets to be cached forever, and the admin can force a cache clear by restarting the server after updating the source code (or using the "Clear server cache" in debug tools - to be added later :-))